### ISSUE TITLE
A11y_SQL Azure Data Studio_NewTable_General_Orientation:All the labels under general tab item in orientation are truncated. 

### ISSUE THREAD

**Issue by yoginigorle** [microsoft/azuredatastudio#19433]
#### GitHub Tags:
#A11yMAS;#A11yTCS;#Win10;#WCAG1.3.4;#DesktopApp;#A11ySev2;#SH_SQL Azure Data Studio_May2022;#SQL Azure Data Studio;

##Environmental Details:
Version: 1.37.0-insider (user setup)
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0

## OS Details :
Microsoft Windows 10

## Repro Steps:
1.Add connection to the server with appropriate credentials.
2.Tab till Servers in left navigation Bar.
3.Tab till tables and Press enter
4.Tab till New tables and press enter
5.Tab till general and press enter.
6.Check all the labels under general tab item in orientation are truncated or not.

## Actual results:
All the labels under general tab item in orientation are truncated. 

## Expected results:
All the labels under general tab item Should be visible properly in orientation. 

## User Impact:
users not able to see and interact with the labels. 

## Attachments:

**Comment by Vishnuvarthancr7** [comment]
#Closed

**Comment by ManishaC1** [comment]
Github tags:#Closed;
